### ISSUE TITLE
GoPro Hero 3 - iPhone 4S not connecting

### ISSUE THREAD

**Issue by seb-1204** [mapillary/mapillary_issues#1897]
I've just tired to connect my GoPro Hero3+ Silver Edition (Version HD3.10.03.02) to an iPhone 4S running iOS 9.1.

1. I connect to the GoPro wifi and start the Mapillary (Version 4.3.1) app. 
1. I go to camera, and after a second or so it recognised the camera and ask me if I want to use. OK
1. after another second I get below message
```
Lost connection
Lost connection to the external camera. The capture has stopped. <OK>
```
4. The display then states "External Camera" but the view is frozen. I actually see a photo which is not the current GoPro or phone camera! (so the photo I've seen is was not taken when they connect)

**Comment by millenbop** [comment]
Thanks for reporting. I will look into this as soon as we get back our Hero3, it is out on loan currently.

**Comment by millenbop** [comment]
I got our Hero3 back today and have looked at it now. 

If you get the error that it disconnects, it probably means you have entered the wrong password. Sorry for not providing a better error description, but the HTTP GET API for Hero3 isn't the best and I cannot distinguish the two.

I found some other bugs though, and they are fixed now as well. Fix will be out in the next release.

Thanks for reporting.
